pymc3/distributions/continuous.py

```diff
@@ -3388,45 +3341,18 @@ class Triangular(BoundedContinuous):
         Upper limit.
     """
 
-    def __init__(self, lower=0, upper=1, c=0.5, *args, **kwargs):
-        self.median = self.mean = self.c = c = at.as_tensor_variable(floatX(c))
-        self.lower = lower = at.as_tensor_variable(floatX(lower))
-        self.upper = upper = at.as_tensor_variable(floatX(upper))
-
-        super().__init__(lower=lower, upper=upper, *args, **kwargs)
-
-    def random(self, point=None, size=None):
-        """
-        Draw random values from Triangular distribution.
+    rv_op = triangular
+    bound_args_indices = [0, 2]  # lower, upper
```

Comment:
Why `2`? And shouldn't these be tuples instead of lists?

Reply:
Changed to tuples. 

The arguments of the triangular RandomOp are in this order `(lower, mode, upper)`, and so we need to specify that the first and last are the ones relevant for the transformation, hence `(0, 2)` (zero-based index)